src/Simplex/Messaging/Notifications/Server/Main.hs

```diff
@@ -71,12 +71,13 @@ ntfServerCLI cfgPath logPath =
           \# and restoring it when the server is started.\n\
           \# Log is compacted on start (deleted objects are removed).\n"
             <> ("enable: " <> onOff enableStoreLog <> "\n\n")
-            <> "log_stats: off\n\n"
-            <> "[TRANSPORT]\n"
-            <> "# host is only used to print server address on start\n"
+            <> "log_stats: off\n\n\
+               \[TRANSPORT]\n\
+               \# host is only used to print server address on start\n"
```

Comment:
this seems more error prone - in addition to \n you have to not forget slashes both in beginning and end, but only on specific lines. I don't think it's a good use for multiline string, concatenation seems more straightforward here.

why change this at all if it's working, if it's not tested miss a symbol and it may fail for new installations

Reply:
we may have tests for this now though as of recently, still

Reply:
I don't see how concatenation is any less error-prone, seems the opposite. In one case you need to type `", eol, <,>, "`, in another: `\, eol, \` - I am not sure how 5 characters are less error-prone, in both cases it either compiles or it doesn't, in either case if you miss `\n` it will not break the line in the output.